Doppler.AccountPlans/Model/Promotion.cs

```diff
@@ -0,0 +1,8 @@
+namespace Doppler.AccountPlans.Model
+{
+    public class Promotion
+    {
+        public int? ExtraCredits { get; set; }
+        public int? DiscountPlanFee { get; set; }
```

Comment:
What do you think about making it explicit?

```suggestion
        public int? DiscountPlanFeePercentage { get; set; }
```

Reply:
Yes is possible because this property is a percentage, I'll do
CC: @jhoffman-ms

Reply:
Done, the name is `DiscountPercentage `